.gitignore

```diff
@@ -13,3 +13,5 @@ db.sqlite3
 
 # macOS
 .DS_Store
+
+input/*.jpg
```

Comment:
It might be a good idea to add an entire temp directory to this gitignore for all of your sample and test images.